src/box/alter.cc

```diff
@@ -5868,68 +5868,67 @@ on_replace_dd_func_index(struct trigger *trigger, void *event)
 	return 0;
 }
 
-struct trigger alter_space_on_replace_space = {
-	RLIST_LINK_INITIALIZER, on_replace_dd_space, NULL, NULL
-};
+struct trigger alter_space_on_replace_space;
```

Comment:
Initializing a struct statically is convenient. Why not add `TRIGGER_INITIALIZER` macro?

Reply:
I haven't thought of that. Fixed. Looks better, indeed.